### ISSUE TITLE
Webshop: chevron/carrot should have role element, div which toggles should have aria-expaned.

### ISSUE THREAD

**Issue by jamalv** [teamforus/forus#2231]
In de header van de website staat een icoon van een chevron dat aangeeft dat hier een verborgen sectie in zit. Deze knop is gemaakt van een niet-interactief div-element. 

De voorleessoftware weet niet dat dit een knop is en wat de status van de verborgen sectie is.

Deze knop dient een role=”button” te krijgen. De status van het verborgen deel kan met aria-expanded worden aangegeven.

![image.png](https://images.zenhubusercontent.com/5ba4ae539f1e9e096ebdcc88/49a800d4-2e70-43ee-910f-fc3d3294f386)

**Comment by GerbenBosschieter** [comment]
Checked and passed on Release/v0.24.0
